include/Gaffer/Animation.h

```diff
@@ -171,8 +171,10 @@ class GAFFER_API Animation : public ComputeNode
 				/// removeActiveClashing is true that key will be removed from the curve and returned
 				/// otherwise that key will remain parented to the curve, become inactive and be returned.
 				/// The new key will be the active key at its time.
+				/// When inheritInterpolation is true the key will inherit interpolation from previous key
+				/// or first key of the curve.
 				/// \undoable
-				KeyPtr addKey( const KeyPtr &key, bool removeActiveClashing = true );
+				KeyPtr addKey( const KeyPtr &key, bool inheritInterpolation = false, bool removeActiveClashing = true );
```

Comment:
When would you use `addKey( inheritInterpolation = True )` rather than `insertKey()`? If the caller of `addKey()` is responsible for setting tangents, won't they need to be aware of the interpolation anyway? I guess what I'm wondering is, for example, what will the NumericPlugValueWidget code look like when we introduce the tangents?